main.py

```diff
@@ -1,6 +1,6 @@
 from module.process import download
 
```

Comment:
Excellent work 🥇.... But there are a few problems:
1. The progress bar work but it does not write the download to the file. As in after the video is indicated by the progress bar as done, the total size of the video file is just 1KB which is not suppose to be so same for the course materials.
2. When a download is ongoing and you click on the download btn again, then some fuzzy clitches happens on progress bar
3. Try and beautify the UI a bit

Overall that is a great work you have done 👍

Reply:
About 1. I believe when I tested it, the file was downloaded properly, I'll recheck that.
The 2nd one the download button is supposed to be disabled when a download is ongoing, if its not then I have to fix that
3. I will try to give it some styles, but I'm not very good at the frontend.